frontend/public/module/k8s/pods.ts

```diff
@@ -234,11 +234,17 @@ export const podPhase = (pod: PodKind): PodPhase => {
   let phase = pod.status.phase || pod.status.reason;
 
   _.each(pod.status.initContainerStatuses, (container: ContainerStatus, i: number) => {
-    const { terminated, waiting } = container.state;
+    const { terminated, waiting, running } = container.state;
+    const initContainerSpec = pod.spec.initContainers.find((c) => c.name === container.name);
+
     if (terminated && terminated.exitCode === 0) {
       return true;
     }
 
+    if (running && container.started && initContainerSpec?.restartPolicy) {
```

Comment:
The `restartPolicy` property allows `Always` only. So I think it looks cleaner not to assign it. WDYT?